public/js/lib/devtools/client/shared/components/reps/window.js

```diff
@@ -41,14 +42,22 @@ define(function (require, exports, module) {
       return getURLDisplayString(grip.preview.url);
     },
 
+    getDisplayValue: function (grip) {
+      if (this.props.mode === "tiny") {
+        return grip.isGlobal ? "Global" : "Window";
+      } else {
+        return this.getLocation(grip);
+      }
+    },
```

Comment:
I like that this doesn't change the original behavior, except for the tiny mode.